src/Locale/nl/cake.po

```diff
@@ -1,297 +1,312 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: CakePHP Dutch Translation\n"
-"POT-Creation-Date: 2016-10-09 14:57+0200\n"
-"PO-Revision-Date: 2016-10-09 14:57+0200\n"
+"POT-Creation-Date: 2017-11-29 08:39+0100\n"
+"PO-Revision-Date: 2017-11-29 20:17+0100\n"
 "Last-Translator: Anne <annuh@users.noreply.github.com>\n"
 "Language-Team: \n"
 "Language: nl\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=n != 1;\n"
-"X-Generator: Poedit 1.8.9\n"
+"X-Generator: Poedit 1.8.4\n"
 
-#: Template/Error/error400.ctp:33 Template/Error/error500.ctp:35
+#: src/Template/Error/error400.ctp:40 src/Template/Error/error500.ctp:43
 msgid "Error"
 msgstr "Fout"
 
-#: Template/Error/error400.ctp:35
+#: src/Template/Error/error400.ctp:41
 msgid "The requested address {0} was not found on this server."
 msgstr "Het opgegeven pad {0} is niet gevonden op deze server."
 
-#: Template/Error/error500.ctp:33
+#: src/Template/Error/error500.ctp:41
 msgid "An Internal Error Has Occurred"
 msgstr "Er is een interne fout opgetreden"
 
-#: Controller/Component/AuthComponent.php:461
+#: vendor/cakephp/cakephp/src/Controller/Component/AuthComponent.php:496
 msgid "You are not authorized to access that location."
 msgstr "U heeft geen toestemming om die locatie te bekijken."
 
-#: Controller/Component/CsrfComponent.php:154
+#: vendor/cakephp/cakephp/src/Controller/Component/CsrfComponent.php:157
+#: vendor/cakephp/cakephp/src/Http/Middleware/CsrfProtectionMiddleware.php:190
 msgid "Missing CSRF token cookie"
-msgstr ""
+msgstr "CSRF token cookie mist"
 
-#: Controller/Component/CsrfComponent.php:158
+#: vendor/cakephp/cakephp/src/Controller/Component/CsrfComponent.php:161
+#: vendor/cakephp/cakephp/src/Http/Middleware/CsrfProtectionMiddleware.php:194
 msgid "CSRF token mismatch."
-msgstr ""
+msgstr "CSRF komt niet overeen."
 
-#: Error/ExceptionRenderer.php:251
+#: vendor/cakephp/cakephp/src/Error/ExceptionRenderer.php:258
 msgid "Not Found"
 msgstr "Niet gevonden"
 
-#: Error/ExceptionRenderer.php:253
+#: vendor/cakephp/cakephp/src/Error/ExceptionRenderer.php:260
 msgid "An Internal Error Has Occurred."
 msgstr "Er is een interne fout opgetreden."
 
-#: I18n/Number.php:90
+#: vendor/cakephp/cakephp/src/Http/Response.php:2400
+msgid "The requested file contains `..` and will not be read."
+msgstr "Het opgevraagde bestand bevat `..` en zal daarom niet worden geopent."
+
+#: vendor/cakephp/cakephp/src/Http/Response.php:2411
+msgid "The requested file was not found"
+msgstr "Het gevraagde bestand werd niet gevonden"
+
+#: vendor/cakephp/cakephp/src/I18n/Number.php:90
 msgid "{0,number,#,###.##} KB"
-msgstr ""
+msgstr "{0,number,#,###.##} KB"
```

Comment:
Why filling it out if not translated (same content)?

Reply:
Actually the comma and dots may be replaced, @WyriHaximus can you enlighten us?